client/components/remove-button/index.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import assign from 'lodash/object/assign';
+import classNames from 'classnames';
+import noop from 'lodash/utility/noop';
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+
+	displayName: 'RemoveButton',
+
+	propTypes: {
+		icon: React.PropTypes.string,
+		disabled: React.PropTypes.bool,
+		onClick: React.PropTypes.func
+	},
+
+	getDefaultProps() {
+		return {
+			icon: 'remove',
+			disabled: false,
+			compact: false,
+			scary: false,
+			onClick: noop
+		};
+	},
+
+	getIconAssignments(){
+
+	},
+
+	render() {
+		const element = 'button';
+		const buttonIcons = {
+			'remove': 		'cross',
+			'trash': 			'trash',
+			'disconnect':	'link-break',
+			'deactivate':	'cross',
+			'delete':			'trash',
+		};
```

Comment:
The formatting is off in this section with tab width set to both 4 and 8 spaces. We should probably either leave a single space after the `:` or align the values vertically.